src/helpers/__tests__/utils.test.ts

```diff
@@ -380,6 +380,13 @@ describe('utils', () => {
         'github.com/datadog/test.git'
       )
       expect(ciUtils.filterAndFormatGithubRemote('github.com/datadog/test.git')).toEqual('github.com/datadog/test.git')
+      expect(ciUtils.filterAndFormatGithubRemote('https://git.some.domain.com:8080/datadog/test.git')).toEqual(
+        'git.some.domain.com/datadog/test.git'
+      )
+      expect(ciUtils.filterAndFormatGithubRemote('git@test.domain.com:datadog/test.git')).toEqual(
+        'test.domain.com/datadog/test.git'
+      )
+      expect(ciUtils.filterAndFormatGithubRemote('example.domain.com/datadog/test.git')).toEqual('example.domain.com/datadog/test.git')
```

Comment:
If we are formatting all git domains, shouldn't we also change the function name (and jsdocs) to be something different than `filterAndFormatGithubRemote`? Maybe just `filterAndFormatGitRemote`? Just a nit, but would be better to keep consistency.

Reply:
@AndrewChubatiuk good point!

Can you add the original `filterAndFormatGithubRemote()` function back (with its old implementation and unit tests) and mark it as deprecated like [that function](https://github.com/andrewchubatiuk/datadog-ci/blob/0e2e9fac8e0c20502245b1cb2898e6f763d7bae1/src/helpers/utils.ts#L100)?

Thanks in advance, and sorry for the delay!

Reply:
Nice, indeed the unit test did not change, so the new implementation did not change in behavior for the old supported cases.